package.json

```diff
@@ -188,11 +188,11 @@
     "node-fetch": "^2.7.0",
     "node-nats-streaming": "^0.3.2",
     "pg": "^8.11.5",
-    "pino": "^8.21.0",
+    "pino": "^9.0.0",
     "pino-http": "^9.0.0",
     "pino-pretty": "^10.0.0",
-    "pino-v6": "npm:pino@^6.13.3",
     "pino-v7": "npm:pino@^7.8.1",
```

Comment:
Should we retain pino version 7, two version checks are sufficient?

Reply:
We can do that, but then I also have to remove pino-pretty. We kept it for testing different transports, but I guess its not important. I will try

Reply:
Done